functions/http/pom.xml

```diff
@@ -61,11 +61,15 @@
       <artifactId>gson</artifactId>
       <version>2.8.6</version>
     </dependency>
+    <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>javax.servlet-api</artifactId>
+      <version>4.0.1</version>
```

Comment:
what is this used for? in generally we should have no dependency on the servlet api

Reply:
And it shouldn't be 4.* anyway, unless you are using a 4.* container.

Reply:
preferably we don't have this at all.

Reply:
We don't. It's been deleted.